src/translations.js

```diff
@@ -84,6 +87,7 @@ const statCardLocales = ({ name, apostrophe }) => {
     "statcard.prs": {
       ar: "مجموع طلبات السحب",
       cn: "拉取请求数（PR）",
+      "zh-tw": "拉取請求數（PR）",
```

Comment:
@ItzMiracleOwO Thanks a lot for your contribution. I don't speak traditional Chinese, but the translations look good in google translate. Is there a language syntax reason you added an extra space at the end of the `statcard.prs` and `statcard.commits`. If not, please remove these extra spaces to merge this pr.

Reply:
Hi
Just adding spaces before and after brackets so that the words would seemed seperated on the card, we usually do the same in documents to make it look better

Reply:
@ItzMiracleOwO Ah, I see. Thanks a lot for the explanation. In that case, the PR looks good to me. Thanks a lot for your contribution.